### DISCUSSION TITLE
Move months independently in mode range with custom caption

### DISCUSSION THREAD

**Discussion by clemeleon** [gpbl/react-day-picker#1974] [Support] [answered]
Hello, I am using numberOfMonths={2} with custom caption and I get the previous and next button on each month. Is there a way to make the buttons work independently meaning when I change the previous month on the calendar the next month should keep the current month to stay in November. Now the behaviour is when I click previous or next, but months move to the next / previous month. IS there a way to achieve this? Thanks

<img width="458" alt="image" src="https://github.com/gpbl/react-day-picker/assets/10097131/0f8446a7-8f31-4413-ac8f-a95a438a61d8">

https://codesandbox.io/p/sandbox/date-picker-w6l7s8

**Comment by gpbl** [comment, marked as answer]
@clemeleon yeah this is a current limitation in DayPicker. We will fix it in the next version!

**Comment by clemeleon** [comment]
@gpbl Thanks for the for the info and yes at the moment I just use two calendar to get that to work, Will look forward to the next version.
